cfbs/commands.py

```diff
@@ -513,6 +519,38 @@ def build_step(module, step, max_length):
             touch(dst)
         assert os.path.isfile(dst)
         sh(f"cat '{src}' >> '{dst}'")
+    elif operation == "directory":
+        src, dst = args
+        if dst in [".", "./"]:
+            dst = ""
+        print(f"{prefix} directory '{src}' 'masterfiles/{dst}'")
+        dstarg = dst  # save this for adding .cf files to inputs
+        src, dst = os.path.join(source, src), os.path.join(destination, dst)
+        defjson = os.path.join(destination, "def.json")
+        merged = read_json(defjson)
+        if not merged:
+            merged = {}
+        if "classes" not in merged:
+            merged["classes"] = {}
+            if "services_autorun_bundles" not in merged["classes"]:
+                merged["classes"]["services_autorun_bundles"] = ["any"]
```

Comment:
This `if` is badly indented/nested.

Reply:
You are right! I was only testing with def.json empty, not already having a classes key :+1: